turbo/transactions/call.go

```diff
@@ -45,6 +48,24 @@ func DoCall(
 			return state, block.Header(), nil
 		}
 	*/
+	num := rpc.LatestBlockNumber
```

Comment:
Can read StageFinish? Probably StageHeaders can be way ahead. Especially after long downtime.